string object_name
string mesh_filepath
geometry_msgs/PoseStamped pose_stamped